src/pages/akcie.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { isFuture } from 'date-fns';
+
+import { Page } from 'components/Page';
+import { FBEventsList } from 'components/FBEventsList';
+import { graphql } from 'gatsby';
+
+export default ({
+  data: {
+    allFacebookEvents: { edges: events },
+  },
+}) => (
+  <Page title="Akcie" url="/akcie">
+    <div className="font-bold text-xl mb-1">Pripravované akcie</div>
```

Comment:
Today's events are listed under past events, I think they should be kept separately

![Screenshot 2019-04-06 at 14 26 55](https://user-images.githubusercontent.com/1255989/55669470-23b69980-5878-11e9-9d6e-437302a4bb2e.png)